test/distribution.js

```diff
@@ -0,0 +1,108 @@
+/* eslint-env mocha */
+
+require('should')
```

Comment:
here i would also do it like this:

```js
const should = require('should')
```

This makes it so that you can write tests like this: `should(nullOrObject).equal(null)`, which is sometimes desirable/useful/the-only-way-to-do-it under certain circumstances (and can give better messages on error)